main.py

```diff
@@ -0,0 +1,108 @@
+import requests
+import config
+from datetime import date, timedelta
+
+
+# --------- GENERAL -----------
+# IFTTT Webhooks credentials, for now
+event_name = ""
+
+# Calculate the start and end dates in YYYY-MM-DD format for the 7-day period,
+# with end_date being yesterday and start_date being 7 days ago.
+today = date.today()
```

Comment:
Maybe put the finding of start dates and end dates in a function, that would increase clarity